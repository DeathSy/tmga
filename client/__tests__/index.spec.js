/* global describe it expect */

import React from 'react'
import renderer from 'react-test-renderer'

import Index from '../pages'

describe('Test Root Component', () => {
  it('should render Home component', () => {
    const index = renderer.create(<Index />)
    const tree = index.toJSON()

    expect(tree).toMatchSnapshot()
  })
})
