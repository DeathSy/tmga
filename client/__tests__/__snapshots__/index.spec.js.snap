// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Root Component should render Home component 1`] = `
<div>
  <div>
    <div>
      <div
        className="ui blue inverted vertical ui left wide visible sidebar menu"
      >
        <div
          className="item"
          onClick={[Function]}
        >
          <i
            aria-hidden="true"
<<<<<<< HEAD
            className="icon"
=======
            className="Logo icon"
>>>>>>> feature/modal
          />
        </div>
        <a
          className="active item"
          onClick={[Function]}
        >
          Home
        </a>
        <a
          className="item"
          onClick={[Function]}
        >
          Timetable Editor
        </a>
        <a
          className="item"
          onClick={[Function]}
        >
          Data Management
        </a>
        <a
          className="item"
          onClick={[Function]}
        >
          Condition Management
        </a>
      </div>
      <h1
        style={
          Object {
            "textAlign": "center",
          }
        }
      >
        Timetable Scheduling Processes
      </h1>
      <table
        className="ui fixed very basic table"
        style={
          Object {
            "margin": "50px",
            "overflow": "scroll",
            "width": "750px",
          }
        }
      >
        <thead
          className=""
          style={
            Object {
              "font-size": "20px",
            }
          }
        >
          <tr
            className=""
          >
            <th
              className="twelve wide"
              style={
                Object {
                  "padding-left": "20px",
                }
              }
            >
              Semester
            </th>
            <th
              className=""
              style={
                Object {
                  "padding-left": "20px",
                }
              }
            >
              Status
            </th>
          </tr>
        </thead>
        <tbody
          className=""
        >
          <tr
            className=""
          >
            <td
              className=""
              style={
                Object {
                  "padding-left": "25px",
                }
              }
            >
              2/2560
            </td>
            <td
              className=""
              style={
                Object {
                  "padding-left": "25px",
                }
              }
            >
               
              <i
                aria-hidden="true"
                className="green checkmark big icon"
              />
            </td>
          </tr>
          <tr
            className=""
          >
            <td
              className=""
              style={
                Object {
                  "padding-left": "25px",
                }
              }
            >
              2/2561
            </td>
            <td
              className=""
              style={
                Object {
                  "padding-left": "25px",
                }
              }
            >
               
              <i
                aria-hidden="true"
                className="red refresh big loading icon"
              />
            </td>
          </tr>
        </tbody>
      </table>
      <button
        className="ui primary button"
        disabled={undefined}
<<<<<<< HEAD
        onClick={[Function]}
=======
        onBlur={[Function]}
        onClick={[Function]}
        onFocus={[Function]}
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
>>>>>>> feature/modal
        role="button"
        style={
          Object {
            "float": "right",
            "marginRight": "62px",
          }
        }
        tabIndex={undefined}
      >
        Create a timetable
      </button>
    </div>
  </div>
</div>
`;
